x-pack/dev-tools/jest/create_jest_config.js

```diff
@@ -38,8 +38,8 @@ export function createJestConfig({ kibanaDirectory, xPackKibanaDirectory }) {
       '^.+\\.html?$': 'jest-raw-loader',
     },
     transformIgnorePatterns: [
-      // ignore all node_modules except @elastic/eui which requires babel transforms to handle dynamic import()
-      '[/\\\\]node_modules(?![\\/\\\\]@elastic[\\/\\\\]eui)[/\\\\].+\\.js$',
+      // ignore all node_modules except @elastic/eui and monaco-editor which requires babel transforms to handle dynamic import()
+      '[/\\\\]node_modules(?![\\/\\\\]@elastic[\\/\\\\]eui)(?![\\/\\\\]monaco-editor)[/\\\\].+\\.js$',
```

Comment:
This is needed since we load monaco and its modules via monaco's ES modules and [jest doesn't natively support ESM yet](https://github.com/facebook/jest/issues/4842)

Reply:
nit: Add this detail to the comment.